lib/hacluster.pm

```diff
@@ -310,8 +311,11 @@ sub get_lun {
 
 sub pre_run_hook {
     my ($self) = @_;
-
-    $prev_console = $autotest::selected_console;
+    if (isotovideo::get_version() == 12){
```

Comment:
Wouldn't it be better with `>= 12` ?

Reply:
Nee, I don't really think that people would like to also manage different versions of the testapi. Plus I can imagine this to be a headache, hence: My suggestion to remove this as soon as a fix is provided, and as soon as osd is deployed